services/dag-indexer/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "dag-indexer",
+  "version": "0.0.1",
+  "description": "SubQuery indexer for indexing DAG nodes from Substrate blockchain",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "subql codegen && subql build",
+    "codegen": "subql codegen",
+    "start:docker": "docker-compose pull && docker-compose up --remove-orphans",
+    "dev": "subql-node --dev",
+    "start": "subql-node -f . --historical false --batch-size 1 --worker-threads 1 --concurrent-blocks 1",
+    "start:slow": "subql-node -f . --historical false --batch-size 5 --worker-threads 1 --concurrent-blocks 1 --fetch-batch-size 3",
+    "start:rate-limited": "subql-node -f . --historical false --batch-size 3 --worker-threads 1 --concurrent-blocks 1 --fetch-batch-size 50 --timeout 60000",
```

Comment:
You will probably want to use our [custom docker image](https://github.com/autonomys/subql/pkgs/container/subql-node-substrate) which allows for more flexible "finalization" handling.